build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -42,11 +49,22 @@ private
 val commitVersionRegex = """(\d+(\.\d+)+)-commit-[a-f0-9]+""".toRegex()
 
 
+/*
+Downloading https://services.gradle.org/distributions-snapshots/gradle-7.5-20220202183149+0000-bin.zip
+Exception in thread "main" java.io.FileNotFoundException:
+https://downloads.gradle-dn.com/distributions-snapshots/gradle-7.5-20220202183149+0000-bin.zip
+ */
```

Comment:
❓ - Does this comment want to say something?

Reply:
To help the reader understand the regex, it wants to tell the reader how the output looks like. But yes, I added one more line explanation in the comment.